Documentation/network/kubernetes/requirements.rst

```diff
@@ -37,10 +37,8 @@ Enable CNI in Kubernetes
 ========================
 
 :term:`CNI` - Container Network Interface is the plugin layer used by Kubernetes to
-delegate networking configuration. CNI must be enabled in your Kubernetes
-cluster in order to install Cilium. This is done by passing
-``--network-plugin=cni`` to kubelet on all nodes. For more information, see
-the `Kubernetes CNI network-plugins documentation <https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/>`_.
+delegate networking configuration. Prior to Kubernetes 1.24, the CNI plugins could also be managed by the kubelet using the ``cni-bin-dir`` and ``network-plugin`` command-line parameters. These command-line parameters were removed in Kubernetes 1.24, with management of the CNI no longer in scope for kubelet.
+ For more information, see the `Kubernetes CNI network-plugins documentation <https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/>`_. To enable the cilium CNI plugin you need to configure the CNI plugin located at ``/etc/cni/net.d/`` . you can do this by making a config file for at the designated location . for making a config file you can refer to : `cilium config documentation <https://docs.cilium.io/en/stable/configuration/>`_. 
```

Comment:
A few things here:

- Let's keep present tense for how things work on K8s 1.24, given that Cilium still supports this version.
- It's not clear what “also” refers to. I think you copy-pasted this sentence from K8s's docs? Please let's adjust it to fit in the context.
- Mind the indentation (` For more information, ...`), the unexpected space at the beginning of the line breaks the build.
- Please fix the case and punctuation.
- Please wrap the lines, keep them under 80 characters in width or so.
- The internal reference (for configuration) needs to be fixed.

```suggestion
delegate networking configuration:

- Up to Kubernetes 1.23, you can enable the CNI in your cluster by passing ``--network-plugin=cni`` to Kubelet on all nodes.
- Starting from Kubernetes 1.24, this command-line parameter has been removed and kubelet no longer manages the CNI. For more information, see the `Kubernetes CNI network-plugins documentation`_. To enable the Cilium CNI plugin, configure the CNI plugin located at ``/etc/cni/net.d/``, by making a configuration file at the designated location. For more information about creating a configuration file, refer to :ref:`Cilium configuration <configuration>`.

.. _Kubernetes CNI network-plugins documentation: https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/
```